package-lock.json

```diff
@@ -1,8 +1,11834 @@
 {
   "name": "boilerplate",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
```

Comment:
Something looks off here: I wouldn't expect to see a "packages" property on the `package-lock.json` object, and it looks like we've duplicated our package lock descriptions. Is it possible that we added `d3` with some additional params?